home-open-api/pom.xml

```diff
@@ -63,13 +63,11 @@
                     <validateOperationDescription>true</validateOperationDescription>
                     <validateOperationTag>true</validateOperationTag>
                     <validateOperationSummary>true</validateOperationSummary>
-                    <validateModelPropertiesExample>true</validateModelPropertiesExample>
+                    <validateModelPropertiesExample>false</validateModelPropertiesExample>
```

Comment:
Why did we disable it? I think we must keep it enabled.

Reply:
Changed to "true".